dist/Renderer.js

```diff
@@ -0,0 +1,14 @@
+class Renderer{
+    renderExpenses(data){
+      const source = $('#expenses-template').html()
+      const template = Handlebars.compile(source)
+      const expense = template({data})
+      $('.expenses-container').empty().append(expense )
+    }
+    // renderAdd(newExpense){
```

Comment:
please delete any unnecessary comments when you push your code a final push to gitHub